cmd/kyma/test/common.go

```diff
@@ -42,3 +44,20 @@ func GetNumberOfFinishedTests(testSuite *oct.ClusterTestSuite) int {
 	}
 	return result
 }
+
+func ListTestSuitesByName(cli octopus.OctopusInterface, names []string) ([]oct.ClusterTestSuite, error) {
+	suites, err := cli.ListTestSuites(metav1.ListOptions{})
+	if err != nil {
+		return nil, errors.Wrap(err, "Unable to list test suites")
+	}
+
+	result := []oct.ClusterTestSuite{}
+	for _, suite := range suites.Items {
+		for _, tName := range names {
+			if suite.ObjectMeta.Name == tName {
+				result = append(result, suite)
```

Comment:
Once we found a match, we do not need to continue iterating the inner loop, we can jump to the next item.

Reply:
I didn't wrote that code I only copied it from here: https://github.com/kyma-project/cli/blame/cb9b22473c8bb49328a2fd2e09ff8521b24d58f0/cmd/kyma/test/status/cmd.go#L196-L211

for DRY.

but I've also added the indexing the names  as we want to optimize it

Reply:
I see... that one is definitely wrong as well.

But then new approach is quite good.